src/components/Request/FormFields/Supervisor.tsx

```diff
@@ -0,0 +1,49 @@
+import { Label, Text } from "@fluentui/react-components";
+import { ContactIcon } from "@fluentui/react-icons-mdl2";
+import { Controller } from "react-hook-form";
+import "components/Request/Request.css";
+import { FormField } from "components/Request/NewRequestForm";
+import { PeoplePicker } from "components/PeoplePicker/PeoplePicker";
+
+const Supervisor = ({ name, form }: FormField) => {
+  return (
+    <div className="requestFieldContainer">
+      <Label
+        id={name + "Id"}
+        size="small"
+        weight="semibold"
+        className="requestFieldLabel"
```

Comment:
I think the Label should have a required attribute, and the Controller a required rule